src/peakrdl_python/templates/addrmap.py.jinja

```diff
@@ -155,6 +163,34 @@ class {{get_fully_qualified_type_name(node)}}_cls(AddressMap):
         return self.__{{child_node.inst_name}}
             {%- endif %}
         {% endfor %}
+
+    {{ child_getter(node, "get_registers",   "Reg",        systemrdlRegNode) }}
```

Comment:
@mpriestleyidex 
The field version has a separate interator for readable_registers and writable_registers. When striding an address map I often want to operate on these separately depending on what is happening

Reply:
I agree - the generic dumping / checking functions I have in mind are normally looking for readable registers.

Reply:
I have made a separate `readable_registers` and `writeable_registers` getter.